src/style/typography.js

```diff
@@ -1,64 +1,75 @@
 import _ from 'lodash';
 import {Constants} from '../helpers';
 
+const WEIGHT_TYPES = {
+  SLIM: '100',
```

Comment:
I don't it's called SLIM, I think it's ultra light, in our internal lib I called it ULIGHT